modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -302,43 +252,107 @@ private String getSwitchToken(String organizationId, String token, HttpClient cl
         EntityUtils.consume(response.getEntity());
         JSONParser parser = new JSONParser();
         org.json.simple.JSONObject json = (org.json.simple.JSONObject) parser.parse(responseString);
-        if (json == null) {
-            throw new Exception("Error occurred while getting the response.");
-        }
```

Comment:
shall we keep this check to avoid any NPEs?

Reply:
Yes, looks better. Instead of throwing an exception, will have an assertNotNull check

Reply:
Fixed by - https://github.com/wso2/product-is/pull/18343/commits/58a99f27bb71ed44eebf9af982e8a143246d0935